src/frontends/tensorflow/src/op/conv_3d_backprop.cpp

```diff
@@ -14,91 +14,95 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_conv_3d_backprop_input_v2_op(const NodeContext& node) {
-    auto ng_filter = node.get_input(1);
-    auto ng_out_backprop = node.get_input(2);
+    TENSORFLOW_OP_VALIDATION(node, node.get_input_size() >= 2, "Conv3DBackpropInput must have at least three inputs.");
+    auto input_sizes = node.get_input(0);
+    auto filter = node.get_input(1);
+    auto out_backprop = node.get_input(2);
 
-    // retrieve attributes for Conv3DBackpropInputV2
+    // retrieve attributes for Conv3DBackpropInput
     auto tf_strides = node.get_attribute<std::vector<int64_t>>("strides");
     auto tf_padding_type = node.get_attribute<std::string>("padding");
+    ov::op::PadType auto_pad = convert_deconv_tf_padding(node, tf_padding_type);
 
     // retrieve optional attributes
-    auto tf_data_format = node.get_attribute<std::string>("data_format", "NDHWC");
     auto tf_dilations = node.get_attribute<std::vector<int64_t>>("dilations", {1, 1, 1, 1, 1});
+    auto tf_explicit_paddings = node.get_attribute<std::vector<int64_t>>("explicit_paddings", {});
+    auto tf_data_format = node.get_attribute<std::string>("data_format", "NDHWC");
 
     TENSORFLOW_OP_VALIDATION(node,
                              tf_data_format == "NDHWC" || tf_data_format == "NCDHW",
-                             "Conv3DBackpropInputV2 data format is neither NDHWC nor NCDHW. "
-                             "Provided data format: ",
-                             tf_data_format);
+                             "Conv3DBackpropInput data format is neither NDHWC nor NCDHW");
+    TENSORFLOW_OP_VALIDATION(node,
```

Comment:
is it mistake when tf_explicit_paddings is set, but auto_pad type is not Explicit? or we can ignore these values?
maybe it's better to check `auto_pad == ov::op::PadType::EXPLICIT && tf_explicit_paddings.size() == 10`

Reply:
let's say `auto_pad = Explicit`, so `auto_pad != ov::op::PadType::EXPLICIT` -> false
will it throw exception in this case? and it doesn't matter what tf_explicit_paddings.size() is actually is.

so maybe it's better:
if (auto_pad == ov::op::PadType::EXPLICIT)
    TENSORFLOW_OP_VALIDATION(..., tf_explicit_paddings.size() == 10, ...)

Reply:
Agree

Reply:
Fixed. Thanks.